src/wallet/wallet.cpp

```diff
@@ -4221,8 +4221,11 @@ std::shared_ptr<CWallet> CWallet::CreateWalletFromFile(interfaces::Chain& chain,
     {
         //We can't rescan beyond non-pruned blocks, stop and throw an error
         //this might happen if a user uses an old wallet within a pruned node
```

Comment:
Nit: space after // in lines 4222 and 4223

Reply:
Perhaps out of PR scope but the docs in lines 4222 and 4223 could be clearer, e.g. is "error" the end of a sentence and "this" the start of a sentence, or...

Reply:
This is only a dev comment, so no need to over-optimize. Will take specific suggestions if I need to touch this pull for other reasons, though.

Reply:
I don't think "no need to over-optimize" is a helpful response. The grammar is wrong. Why not correct it?

Reply:
Added a dot and space if that is what you meant